skipper.go

```diff
@@ -310,6 +309,18 @@ type Options struct {
 	// Custom data clients to be used together with the default etcd and Innkeeper.
 	CustomDataClients []routing.DataClient
 
+	// CustomHttpHandlerWrap provides ability to wrap http.Handler created by skipper.
+	// http.Handler is used for accepting incoming http requests.
+	// It allows to add additional logic (for example tracing) by providing a wrapper function
+	// which accepts original skipper handler as an argument and returns a wrapped handler
+	CustomHttpHandlerWrap func(http.Handler) http.Handler
```

Comment:
Maybe it's then better to call it `WithHttpHandler`?
@aryszka do you have a name suggestion?

Reply:
I would expect `WithHttpHandler` to take just http.Handler object aka signature:

```
WithHttpHandler http.Handler
```

I don't like the naming part myself but suggestions are welcome ;)

Reply:
I would drop the "Custom..." prefix, and just would call it WrapHandler and WrapTransport, but this is also fine with me.

Reply:
The reason I put `Custom` in the name is because currently there are CustomPredicates, CustomFilters and those are things that you pass in as structs (interfaces) instead of cli-arguments.